### ISSUE TITLE
include typings into package.json

### ISSUE THREAD

**Issue by aperdomob** [mulesoft-labs/raml-generator#20]
In order to avoid any types in other project please add typings into `package.json`

http://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

**Comment by sichvoge** [comment]
I think, with TypeScript 2.x, typings is deprecated already, isn't it?

**Comment by aperdomob** [comment]
yeah, the library typings is deprecated but I refer to key into package.json, this key contains  the path where are your definitions for example...
```
"typings": "dist/declarations/src/index.d.ts"
```

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

**Comment by DaSchTour** [comment]
Had the same issues. #21 should fix it
